### PR TITLE
Added homepage and model annotations

### PR DESCRIPTION
**Pull request by jvsa1102**
Annotate gem installed and used
Homepage and pages controller added

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -1,3 +1,14 @@
+# == Schema Information
```
Comment (DarlanSel) [inline app/models/user.rb:1]: I recomend to configure the annotate gem to annotate at the end of the file. Keep the file header cleaner

File: app/controllers/pages_controller.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+class PagesController < ApplicationController
+  def home
+  end
+end
```
Comment (DarlanSel) [inline app/controllers/pages_controller.rb:4]: Missing new line. Github complaining

File: app/views/pages/home.html.erb [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>CommUnity</title>
+</head>
+<body>
+  <h1>Welcome to CommUnity</h1>
+  <%= link_to "Register", new_user_path(@user)%>
+</body>
+</html>
```
Comment (DarlanSel) [inline app/views/pages/home.html.erb:13]: Missing new line

File: app/views/pages/home.html.erb [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>CommUnity</title>
+</head>
+<body>
+  <h1>Welcome to CommUnity</h1>
+  <%= link_to "Register", new_user_path(@user)%>
```
Comment (DarlanSel) [inline app/views/pages/home.html.erb:11]: Prefer to use single quote

File: app/views/users/new.html.erb [hunk 5]
```diff
@@ -1,9 +1,9 @@
-<h1>New user</h1>
+<h1>Register</h1>
 
 <%= render "form", user: @user %>
 
 <br>
 
 <div>
-  <%= link_to "Back to users", users_path %>
+  <%= link_to "Back to home", root_path %>
```
Comment (DarlanSel) [inline app/views/users/new.html.erb:8]: Prefer to use single quote

File: app/controllers/pages_controller.rb (hunk 2 above)
Comment (DarlanSel) [reply app/controllers/pages_controller.rb:4]: If you go to `Files changed` section you will see the complain:
image

File: app/models/user.rb (hunk 1 above)
Comment (jvsa1102) [reply app/models/user.rb:8]: Fixed

File: app/controllers/pages_controller.rb (hunk 2 above)
Comment (jvsa1102) [reply app/controllers/pages_controller.rb]: Fixed

File: app/views/pages/home.html.erb (hunk 4 above)
Comment (jvsa1102) [reply app/views/pages/home.html.erb]: Fixed

File: app/views/users/new.html.erb (hunk 5 above)
Comment (jvsa1102) [reply app/views/users/new.html.erb]: Fixed

**Pull request merged**